descriptors/ModuleDescriptor-template.json

```diff
@@ -113,6 +121,22 @@
         }
       ]
     },
+    {
+      "id": "remote-storage-check-in",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["POST"],
+          "pathPattern": "/remote-storage/retrieve/{remoteStorageConfigurationId}/checkInItem",
+          "permissionsRequired" : ["remote-storage.check-in.item.post"],
+          "modulePermissions" : [
+            "inventory-storage.locations.item.get",
+            "circulation.check-in-by-barcode.post",
+            "modperms.circulation.check-in-by-barcode.post"
```

Comment:
This is internal permissions for mod-circulation https://github.com/folio-org/mod-circulation/blob/8c6583b094fd58eaa38347c39b696bc7f2e99bb9/descriptors/ModuleDescriptor-template.json#L187. We only need "inventory-storage.locations.item.get" and "circulation.check-in-by-barcode.post here.

Reply:
Unnecessary permission was removed.